CHANGELOG.MD

```diff
@@ -1,3 +1,22 @@
+## 1.289.1 (2024-04-17)
+
+### Features
+
+* Update golang.org/x/sys version to 0.1.0.
+* Update gopkg.in/yaml.v3 version to 3.0.0.
+* Update go version to 1.17
+* Update google.golang.org/protobuf version to 1.33.0.
+
+## 1.279.1 (2023-12-21)
+
+### Features
+
+* Improve reading config file
+* Split large spans into multiple export messages
```

Comment:
Spans are never split, only multiple spans are distributed over multiple messages.
Maybe user-facing we could say "Improve reliability when sending large traces"

Reply:
Thanks, I'll use your suggestion.

Note: I was not working on the changes so I don't know what exactly changed , I'm just trying to document based on commits.